### PR TITLE
fix nupkg readme

### PR DESCRIPTION
**Pull request by bruno-garcia**
@Tyrrrz did this work for you when you added it?
Didn't show up on nuget.org

I added it [on another project](https://github.com/bruno-garcia/unoptimized-assembly-detector) and I assume this should do (didn't check here yet)

#skip-changelog

### CODE REVIEW

**Comment by bruno-garcia** [comment]
Looks like the README isn't shown if the changelog is availalble:

https://www.nuget.org/packages/UnoptimizedAssemblyDetector/0.0.5

**Comment by bruno-garcia** [comment]
Turns out the README item isn't in the nuspec

```
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
  <metadata minClientVersion="4.0">
    <id>UnoptimizedAssemblyDetector</id>
    <version>0.0.5</version>
    <authors>Bruno Garcia</authors>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <license type="expression">MIT</license>
    <licenseUrl>https://licenses.nuget.org/MIT</licenseUrl>
    <icon>unoptimized.png</icon>
    <description>Package Description</description>
    <releaseNotes>Changes since version 0.0.4

- Fixed conflict with Microsoft.Build.Utilities.Core
- Added a logo to nuget package and acknowledgment to readme
- CI and release to nuget with craft CLI
Changes since version 0.0.3

- Works also on Visual Studio (.NET Framework msbuild)
- Initial release
--&gt;
      See full changelog at https://github.com/bruno-garcia/unoptimized-assembly-detector/blob/main/CHANGELOG.md</releaseNotes>
    <tags>Performance Diagnostics</tags>
    <repository type="git" url="https://github.com/bruno-garcia/unoptimized-assembly-detector" commit="12293e38473500014fb98b39862f52ea88a4f3b0" />
    <dependencies>
      <group targetFramework=".NETFramework4.6.1">
        <dependency id="System.Reflection.Metadata" version="5.0.0" exclude="Build,Analyzers" />
      </group>
      <group targetFramework=".NETStandard2.0">
        <dependency id="System.Reflection.Metadata" version="5.0.0" exclude="Build,Analyzers" />
      </group>
    </dependencies>
  </metadata>
</package>

```

**Comment by bruno-garcia** [comment]
Oh, does it work only when uploaded through the web UI? wtf

**Comment by Tyrrrz** [comment]
> > @Tyrrrz did this work for you when you added it?
> > Didn't show up on nuget.org
> > I added it [on another project](https://github.com/bruno-garcia/unoptimized-assembly-detector) and I assume this should do (didn't check here yet)
> > #skip-changelog
> 
> No. I asked in [#956 (comment)](https://github.com/getsentry/sentry-dotnet/issues/956#issuecomment-836837475) whether something special needs to be enabled to get it to work, but then we just decided to YOLO it. There is no documentation on this feature anywhere that I could find.

> Oh, does it work only when uploaded through the web UI? wtf

Yeah, that's what I arrived at too, in that comment I linked.

**Comment by bruno-garcia** [comment]
This sucks.

File: src/Directory.Build.props [hunk 1]
```diff
@@ -50,8 +51,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="$(MSBuildThisFileDirectory)/../.assets/sentry-nuget.png" Pack="true" PackagePath=""/>
-    <None Include="$(MSBuildThisFileDirectory)/../README.md" Pack="true" PackagePath=""/>
+    <None Include="$(MSBuildThisFileDirectory)../.assets/sentry-nuget.png" Pack="true" PackagePath=""/>
+    <None Include="$(MSBuildThisFileDirectory)../README.md" Pack="true" PackagePath=""/>
```
Comment (Tyrrrz) [inline src/Directory.Build.props:55]: The `/` shouldn't matter I believe because consequent `/` are usually collapsed (at least on Windows)

**Pull request merged**